package.json

```diff
@@ -58,7 +60,7 @@
     "prettier-plugin-solidity": "^1.0.0-alpha.54"
   },
   "resolutions": {
-    "arb-provider-ethers/ethers": "~4.0.47",
+    "arb-provider-ethers/ethers": "~5.0.24",
```

Comment:
This won't be needed anymore 😄

Reply:
keeping as per https://github.com/OffchainLabs/arbitrum/pull/798#discussion_r585903281 (altho the version update shouldn't be there!)